pages/california.vue

```diff
@@ -104,21 +105,53 @@ export default {
       set(value) {
         this.$store.commit('setZipCode', value)
       }
+    },
+    filteredAssemblyMembers () {
+      if (this.voteFilter) {
+        return this.assemblyMembers.filter(p => p.supports_cra.toString() === this.voteFilter)
+      } else {
+        return this.assemblyMembers
+      }
+    },
+    filteredSenators () {
+      if (this.voteFilter) {
+        return this.senators.filter(p => p.supports_cra.toString() === this.voteFilter)
+      } else {
+        return this.senators
+      }
     }
   },
 
   async asyncData() {
-    let politicians = []
+    let assemblyMembers = []
+    let senators = []
 
     try {
       const { data } = await axios.get('https://data.battleforthenet.com/scoreboard/california.json')
-      politicians = data.filter(p => p.organization === 'Assembly')
+      assemblyMembers = data.filter(p => p.organization === 'Assembly')
+      senators = data.filter(p => p.organization === 'Senate')
```

Comment:
While all members could have been one variable and then filtered on the client side, I thought it might be a slight performance bump not to do extra filtering in browser.